src/applications/simple-forms/21-0845/config/helpers.js

```diff
@@ -4,6 +4,8 @@ import { focusByOrder } from 'platform/utilities/ui';
 export const pageFocusScroll = () => {
   return () => {
     scrollTo('topScrollElement');
-    focusByOrder(['va-segmented-progress-bar', 'h2']);
+    setTimeout(() => {
+      focusByOrder(['va-segmented-progress-bar', 'h2']);
+    }, 100);
```

Comment:
Wrapping in a `setTimeout` seems to fix the SSN input focus issues that was showing up